client/my-sites/earn/home.tsx

```diff
@@ -149,15 +149,14 @@ const Home = () => {
 	 * Return the content to display in the Simple Payments card based on the current plan.
 	 */
 	const getSimplePaymentsCard = (): PromoSectionCardProps => {
-		const supportLink = localizeUrl(
-			'https://wordpress.com/support/wordpress-editor/blocks/pay-with-paypal/'
-		);
 		const cta = hasSimplePayments
 			? {
-					text: translate( 'Learn how to get started' ),
+					text: translate( 'Learn more' ),
 					action: () => {
 						trackCtaButton( 'simple-payments' );
-						page( supportLink );
+						window.location.href = localizeUrl(
+							'https://wordpress.com/support/wordpress-editor/blocks/pay-with-paypal/'
```

Comment:
Would be great to use [support article dialog](https://github.com/Automattic/wp-calypso/tree/6c27c90feb519de845f39819ef58c5e4b728a279/client/blocks/support-article-dialog) here, unless there are any issues with the big doc fitting in smaller window (which will be issue with mobile anyway).

Reply:
Yup, will tackle this in a follow up.